utils/helpers.js

```diff
@@ -162,6 +164,23 @@ const appendUTMParams = (url, spaceId) => {
   return new_url.href;
 }
 
+/**
+ * Calculates whether an object is visible by checking for intersections between
+ * the object's bounding box and the camera's frustum.
+ * @param {min: number[]} bbMin
+ * @param {max: number[]} bbMax
+ * @param {number[]} cameraProjMatrix
+ * @param {number[]} cameraWorldMatrix
+ * @returns
+ */
+const visibilityCheck = (bbMin, bbMax, cameraProjMatrix, cameraWorldMatrix) => {
+  const boundingBox = new Box3(new Vector3().fromArray(bbMin), new Vector3().fromArray(bbMax));
+  const frustum = new Frustum().setFromProjectionMatrix(new Matrix4().fromArray(cameraProjMatrix));
+  frustum.planes.forEach(plane => plane.applyMatrix4(new Matrix4().fromArray(cameraWorldMatrix)));
+  console.log('is visible: ', frustum.intersectsBox(boundingBox));
```

Comment:
Do we want to keep this log?

Reply:
Oh whoops, missed that one, lemme get rid of that.